portfolio/src/main/webapp/coding.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  
+  <title>Portfolio</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Lekton" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+  <link rel ="stylesheet" href = "style.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  </head>
+<body>
+<nav class="navbar navbar-inverse">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#" style="font-family: Lekton">Madeline Cuozzo</a>
+    </div>
+    <ul class="nav navbar-nav">
+      <li class="active" style="font-family: Lekton" ><a href="index.html">About Me</a></li>
+      <li class="dropdown"><a class="dropdown-toggle" data-toggle="dropdown" href="#" style="font-family: Lekton">Hobbies<span class="caret"></span></a>
+        <ul class="dropdown-menu">
+          <li><a href="coding.html" style="font-family: Lekton" >Coding</a></li>
+          <li><a href="#" style="font-family: Lekton">Sports</a></li>
+          <li><a href="#" style="font-family: Lekton">Violin</a></li>
+        </ul>
+      </li>
+      <li><a href="./gallery.html" style="font-family: Lekton">Gallery</a></li>
+      <li><a href="#" style="font-family: Lekton">Contact Me</a></li>
+    </ul>
+  </div>
+</nav>
+
+<h1 style="font-family: Lekton; font-style: bold">Coding</h1>
+
+<div class="row">
+  <div class="column">
+    <img class="myImages" id="myImg" src="../images/Langauges.png" id="langImg" alt=
```

Comment:
IDs are generally not advisable, especially for reused elements. Classes are recommended for reusability.